.evergreen/config.yml

```diff
@@ -39,9 +38,9 @@ tasks:
 
 buildvariants:
 
-  - name: simple-variant
+  - name: linux-rhel
     tags: ["ci"]
-    display_name: Simple Variant
+    display_name: Redhat Linux
```

Comment:
I'd still prefer just Linux until we feel the need to test on multiple linux variants (which we never will !).